### PR TITLE
Simplify the Python version check in `./wpt`

### PR DESCRIPTION
**Pull request by foolip**
This is more similar to our other checks, for example in localpaths.py.

### CODE REVIEW

File: wpt [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 if __name__ == "__main__":
     import sys
-    if (sys.version_info.major < 3 or
-            (sys.version_info.major == 3 and sys.version_info.minor < 6)):
+    if (sys.version_info < (3, 6)):
```
[the remark this answers is not in the document]
Comment (foolip) [reply wpt:5]: I could, but isn't that less readable for people who don't know Python, who haven't though about the fact that commas are what creates tuples? The cases in localpaths.py and transport.py have the parens.

[the remark this answers is not in the document]
Comment (foolip) [reply wpt:5]: (If you don't want them, I'll remove them in more places.)

[the remark this answers is not in the document]
Comment (jgraham) [reply wpt:5]: I just meant `if sys.version_info < (3, 6):`

[the remark this answers is not in the document]
Comment (foolip) [reply wpt:5]: Oh, those parens! Yes of course they should go :)

**Review by jgraham — APPROVED** [review]

**Pull request merged**